src/mscorlib/src/Microsoft/Win32/Win32Native.cs

```diff
@@ -916,12 +916,12 @@ internal static bool DoesWin32MethodExist(String moduleName, String methodName)
         // a web client asking a server for "http://server/com1.aspx" and
         // then causing a worker process to hang.
         [System.Security.SecurityCritical]  // auto-generated
-        internal static SafeFileHandle SafeCreateFile(String lpFileName,
+        internal static Win32NativeCloseHandle SafeCreateFile(String lpFileName,
```

Comment:
Is there code that is getting compiled outside of what is in \mscorlib? I was going to create overloads of these where it apparently mattered (usages from the old FileStream).